netpyne_ui/netpyne_geppetto.py

```diff
@@ -240,6 +265,18 @@ def getAvailableStimSources(self):
     
     def getAvailableSynMech(self):
         return netParams.synMechParams.keys()
+    
+    def getAvailableMechs(self):
+        mechs = utils.mechVarList()['mechs']
+        for key in mechs.keys():
+            if 'ion' in key: del mechs[key]
+        for key in ["morphology", "capacitance", "extracellular"]: del mechs[key]
+        return mechs.keys()
+    
+    def getMechParams(self, mechanism):
+        lenght = len(mechanism) + 1
```

Comment:
typo, it should be lenght... but I think it would be even better to remove this variable